integrations/server/test_covidcast.py

```diff
@@ -311,6 +311,100 @@ def test_location_wildcard(self):
       'message': 'success',
     })
 
+  def test_geo_value(self):
+    """test different variants of geo types: single, *, multi."""
+
+    # insert dummy data
+    self.cur.execute('''
+      insert into covidcast values
+        (0, 'src', 'sig', 'day', 'county', 20200414, '11111',
+          123, 10, 11, 12, 456, 13, 20200414, 0, 1, False),
+        (0, 'src', 'sig', 'day', 'county', 20200414, '22222',
+          123, 20, 21, 22, 456, 23, 20200414, 0, 1, False),
+        (0, 'src', 'sig', 'day', 'county', 20200414, '33333',
+          123, 30, 31, 32, 456, 33, 20200414, 0, 1, False),
+        (0, 'src', 'sig', 'day', 'msa', 20200414, '11111',
+          123, 40, 41, 42, 456, 43, 20200414, 0, 1, False),
+        (0, 'src', 'sig', 'day', 'msa', 20200414, '22222',
+          123, 50, 51, 52, 456, 53, 20200414, 0, 1, False),
+        (0, 'src', 'sig', 'day', 'msa', 20200414, '33333',
+          123, 60, 61, 62, 456, 634, 20200414, 0, 1, False)
+    ''')
+    self.cnx.commit()
+
+    def fetch(geo_value):
+      # make the request
+      params = {
+        'source': 'covidcast',
+        'data_source': 'src',
+        'signal': 'sig',
+        'time_type': 'day',
+        'geo_type': 'county',
+        'time_values': 20200414,
+      }
+      if isinstance(geo_value, list):
+        params['geo_values'] = ','.join(geo_value)
+      else:
+        params['geo_value'] = geo_value
+      response = requests.get(BASE_URL, params=params)
+      response.raise_for_status()
+      response = response.json()
+
+      return response
+
+    counties = [{
+      'time_value': 20200414,
+      'geo_value': '11111',
+      'value': 10,
+      'stderr': 11,
+      'sample_size': 12,
+      'direction': 13,
+      'issue': 20200414,
+      'lag': 0,
+      'signal': 'sig',
+    }, {
+      'time_value': 20200414,
+      'geo_value': '22222',
+      'value': 20,
+      'stderr': 21,
+      'sample_size': 22,
+      'direction': 23,
+      'issue': 20200414,
+      'lag': 0,
+      'signal': 'sig',
+    }, {
+      'time_value': 20200414,
+      'geo_value': '33333',
+      'value': 30,
+      'stderr': 31,
+      'sample_size': 32,
+      'direction': 33,
+      'issue': 20200414,
+      'lag': 0,
+      'signal': 'sig',
+    }]
+
+    r = fetch('*')
```

Comment:
Please add comments to each call to `fetch()` indicating the specific behavior being tested.